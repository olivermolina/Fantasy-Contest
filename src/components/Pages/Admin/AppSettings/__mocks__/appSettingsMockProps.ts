export const appSettingsMockProps = {
  appSettings: {
    BONUS_CREDIT_FREE_ENTRY_EQUIVALENT: '25',
    STAKE_TYPE_FREE_ENTRY: 'ALL_IN,INSURED',
    NUMBER_OF_PLAYERS_FREE_ENTRY: '2,3,4',
    MAX_MATCH_DEPOSIT_AMOUNT: '100',
    DEPOSIT_AMOUNT_OPTIONS: '10,25,50,75,100,200',
    RELOAD_BONUS_TYPE: 'FLAT',
    RELOAD_BONUS_AMOUNT: '0',
    MAX_RETENTION_BONUS: '50',
    RETENTION_BONUS_WEEKLY_CHANCE: '1',
    RETENTION_BONUS_MATCH_MULTIPLIER: '1',
    REFERRAL_CREDIT_AMOUNT: '25',
    REFERRAL_CUSTOM_TEXT:
      'Refer a friend from the link or code in your profile and get $25 in bonus credit!!',
    MIN_BET_AMOUNT: '1',
    MAX_BET_AMOUNT: '50',
    MAX_MARKET_ODDS: '150',
    MIN_MARKET_ODDS: '-150',
    initial: false,
    CHALLENGE_PROMO_MESSAGE:
      'Pick 2-4 players. Predict if they will get MORE or LESS than their projection.',
    REPEAT_ENTRIES_LIMIT: '1',
  },
  onSubmit: () => console.log('onSubmit'),
};
