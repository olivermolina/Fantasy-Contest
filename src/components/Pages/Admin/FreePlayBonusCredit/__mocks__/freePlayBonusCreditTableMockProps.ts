import { PaymentStatusCode } from '~/constants/PaymentStatusCode';
import { TransactionType } from '@prisma/client';

export const freePlayBonusCreditTableMockProps = {
  onSubmitDeletePlayFreeCredit: () => console.log('delete'),
  data: [
    {
      id: '1',
      transactionDate: new Date(),
      amountBonus: 100,
      status: PaymentStatusCode.COMPLETE,
      type: TransactionType.CREDIT,
    },
    {
      id: '2',
      transactionDate: new Date(),
      amountBonus: 100,
      status: PaymentStatusCode.CANCELLED,
      type: TransactionType.DEBIT,
    },
  ],
  totalBonusCreditBalance: 0,
};
