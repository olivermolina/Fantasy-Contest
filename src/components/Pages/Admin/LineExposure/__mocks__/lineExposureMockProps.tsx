import { League } from '@prisma/client';

export const lineExposureMockProps = {
  data: [
    {
      id: '1',
      name: 'LeBron James',
      category: 'Points',
      league: League.NBA,
      status: 'Scheduled',
      matchup: 'LAL @ DEN',
      total: 35,
      gamedate: '2023-01-23',
      gametime: '03:00:00 PM',
      underOdds: 100,
      overOdds: -120,
      overExposure: 4,
      underExposure: 2,
      betLegs: [
        {
          id: 'ticket-id',
          betId: 'bet-id',
          entryDate: '2023-01-22',
          payout: 100,
          stake: 10,
          type: 'OVER_ODDS',
          user: {
            id: 'user-1',
            firstName: 'John',
            lastName: 'Doe',
            username: 'jdoe',
          },
          status: 'WIN',
        },
      ],
    },
  ],
  setDateRange: () => console.log('setDateRange'),
  dateRange: null,
  selectedLeague: League.NBA,
  setSelectedLeague: () => console.log('setSelectedLeague'),
};
