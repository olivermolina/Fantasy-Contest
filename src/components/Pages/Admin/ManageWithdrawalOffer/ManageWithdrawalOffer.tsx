import React, { useEffect } from 'react';
import { Button, InputAdornment, TextField } from '@mui/material';
import { useForm } from 'react-hook-form';
import { yupResolver } from '@hookform/resolvers/yup';
import * as Yup from 'yup';

const InputValidationSchema = Yup.object().shape({
  maxOffer: Yup.number()
    .typeError('Please provide a max bonus credit to offer.')
    .min(1, 'Minimum is $1'),
  matchMultiplier: Yup.number()
    .typeError('Please provide bonus match multiplier')
    .min(1, 'Minimum is 1'),
  weeklyOfferChance: Yup.number()
    .typeError('Please provide how many weekly offers are allowed.')
    .min(1, 'Minimum is 1'),
});

export interface WithdrawalOfferInputs {
  /**
   * Max bonus credit to offer
   * @example 500
   **/
  maxOffer: number;
  /**
   * Bonus match multiplier
   * @example 3
   **/
  matchMultiplier: number;
  /**
   * Withdrawal bonus credit weekly offers are allowed
   * @example 2
   **/
  weeklyOfferChance: number;
}

interface Props extends WithdrawalOfferInputs {
  /**
   * Submit form function
   */
  onSubmit: (inputs: WithdrawalOfferInputs) => void;
}

export default function ManageWithdrawalOffer(props: Props) {
  const { maxOffer, matchMultiplier, weeklyOfferChance, onSubmit } = props;

  const {
    register,
    handleSubmit,
    formState: { errors },
    reset,
  } = useForm<WithdrawalOfferInputs>({
    resolver: yupResolver(InputValidationSchema),
  });

  useEffect(() => {
    reset({
      maxOffer,
      matchMultiplier,
      weeklyOfferChance,
    });
  }, [maxOffer, matchMultiplier, weeklyOfferChance]);

  return (
    <div className={'flex flex-col p-2 max-w-md'}>
      <form onSubmit={handleSubmit(onSubmit)}>
        <div className={`flex flex-col items-start gap-4`}>
          <TextField
            id="outlined-basic"
            label="Max Bonus Credit"
            variant="standard"
            type={'number'}
            fullWidth
            InputProps={{
              startAdornment: (
                <InputAdornment position="start">$</InputAdornment>
              ),
            }}
            {...register('maxOffer')}
            error={!!errors?.maxOffer}
            helperText={errors?.maxOffer?.message}
            size={'small'}
          />

          <TextField
            {...register('matchMultiplier')}
            id="outlined-basic"
            label="Bonus match multiplier"
            variant="standard"
            type={'number'}
            fullWidth
            error={!!errors?.matchMultiplier}
            helperText={errors?.matchMultiplier?.message}
            size={'small'}
            InputProps={{
              startAdornment: (
                <InputAdornment position="start"></InputAdornment>
              ),
            }}
          />

          <TextField
            id="outlined-basic"
            label="Weekly Offers Chances"
            variant="standard"
            type={'number'}
            fullWidth
            {...register('weeklyOfferChance')}
            error={!!errors?.weeklyOfferChance}
            helperText={errors?.weeklyOfferChance?.message}
            size={'small'}
            InputProps={{
              startAdornment: (
                <InputAdornment position="start"></InputAdornment>
              ),
            }}
          />

          <Button variant="contained" type="submit" fullWidth name={'submit'}>
            Submit
          </Button>
        </div>
      </form>
    </div>
  );
}
