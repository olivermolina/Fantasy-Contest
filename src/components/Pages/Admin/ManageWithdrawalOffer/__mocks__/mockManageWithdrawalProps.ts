export const mockManageWithdrawalProps = {
  onSubmit: () => console.log('test'),
  maxOffer: 500,
  matchMultiplier: 1,
  weeklyOfferChance: 2,
};
