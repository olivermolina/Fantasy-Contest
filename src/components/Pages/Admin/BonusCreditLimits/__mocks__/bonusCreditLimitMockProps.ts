import { contestCategoriesMock } from '~/components/Pages/Admin/ManageFreeSquarePromotion/__mocks__/contestCategoriesMock';

export const bonusCreditLimitMockProps = {
  onSubmit: () => console.log('submit'),
  contestCategories: contestCategoriesMock,
  data: {
    numberOfPlayers: ['2', '3', '4'],
    bonusCreditFreeEntryEquivalent: 25,
    stakeType: ['INSURED', 'ALL_IN'],
  },
};
