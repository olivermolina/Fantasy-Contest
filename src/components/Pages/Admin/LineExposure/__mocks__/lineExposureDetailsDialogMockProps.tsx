import { lineExposureMockProps } from './lineExposureMockProps';

export const lineExposureDetailsDialogMockProps = {
  row: lineExposureMockProps.data[0],
  open: false,
  handleClose: () => console.log(close),
};
