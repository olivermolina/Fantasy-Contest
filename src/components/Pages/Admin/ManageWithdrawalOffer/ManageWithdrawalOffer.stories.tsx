import React from 'react';
import { ComponentMeta, ComponentStory } from '@storybook/react';

import WithdrawalOffer from './ManageWithdrawalOffer';
import { mockManageWithdrawalProps } from './__mocks__/mockManageWithdrawalProps';

// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
export default {
  title: 'Lockspread/Admin/Management/FreePlayBonusCredit',
  component: WithdrawalOffer,
  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
  argTypes: {
    backgroundColor: { control: 'color' },
  },
} as ComponentMeta<typeof WithdrawalOffer>;

// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
const Template: ComponentStory<typeof WithdrawalOffer> = (args) => (
  <WithdrawalOffer {...args} />
);

export const ManageWithdrawalOffer = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args

ManageWithdrawalOffer.args = mockManageWithdrawalProps;
