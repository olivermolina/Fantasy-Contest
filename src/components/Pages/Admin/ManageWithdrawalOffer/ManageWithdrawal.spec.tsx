import { render } from '@testing-library/react';
import ManageWithdrawalOffer from './ManageWithdrawalOffer';
import { mockManageWithdrawalProps } from './__mocks__/mockManageWithdrawalProps';

describe('ManageWithdrawalOffer', () => {
  it('should render correctly', () => {
    const container = render(
      <ManageWithdrawalOffer {...mockManageWithdrawalProps} />,
    );
    expect(container).toMatchSnapshot();
  });
});
