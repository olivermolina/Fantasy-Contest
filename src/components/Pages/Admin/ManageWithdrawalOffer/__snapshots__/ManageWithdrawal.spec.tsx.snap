// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManageWithdrawalOffer should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col p-2 max-w-md"
      >
        <form>
          <div
            class="flex flex-col items-start gap-4"
          >
            <div
              class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard css-1c2i806-MuiFormLabel-root-MuiInputLabel-root"
                data-shrink="true"
                for="outlined-basic"
                id="outlined-basic-label"
              >
                Max Bonus Credit
              </label>
              <div
                class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall MuiInputBase-adornedStart css-l4u8b9-MuiInputBase-root-MuiInput-root"
              >
                <div
                  class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-standard MuiInputAdornment-sizeSmall css-ittuaa-MuiInputAdornment-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-1pnmrwp-MuiTypography-root"
                  >
                    $
                  </p>
                </div>
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiInput-input MuiInputBase-inputSizeSmall MuiInputBase-inputAdornedStart css-nz481w-MuiInputBase-input-MuiInput-input"
                  id="outlined-basic"
                  name="maxOffer"
                  type="number"
                  value=""
                />
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard css-1c2i806-MuiFormLabel-root-MuiInputLabel-root"
                data-shrink="true"
                for="outlined-basic"
                id="outlined-basic-label"
              >
                Bonus match multiplier
              </label>
              <div
                class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall MuiInputBase-adornedStart css-l4u8b9-MuiInputBase-root-MuiInput-root"
              >
                <div
                  class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-standard MuiInputAdornment-sizeSmall css-ittuaa-MuiInputAdornment-root"
                >
                  <span
                    class="notranslate"
                  >
                    ​
                  </span>
                </div>
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiInput-input MuiInputBase-inputSizeSmall MuiInputBase-inputAdornedStart css-nz481w-MuiInputBase-input-MuiInput-input"
                  id="outlined-basic"
                  name="matchMultiplier"
                  type="number"
                  value=""
                />
              </div>
            </div>
            <div
              class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard css-1c2i806-MuiFormLabel-root-MuiInputLabel-root"
                data-shrink="true"
                for="outlined-basic"
                id="outlined-basic-label"
              >
                Weekly Offers Chances
              </label>
              <div
                class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall MuiInputBase-adornedStart css-l4u8b9-MuiInputBase-root-MuiInput-root"
              >
                <div
                  class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-standard MuiInputAdornment-sizeSmall css-ittuaa-MuiInputAdornment-root"
                >
                  <span
                    class="notranslate"
                  >
                    ​
                  </span>
                </div>
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiInput-input MuiInputBase-inputSizeSmall MuiInputBase-inputAdornedStart css-nz481w-MuiInputBase-input-MuiInput-input"
                  id="outlined-basic"
                  name="weeklyOfferChance"
                  type="number"
                  value=""
                />
              </div>
            </div>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
              name="submit"
              tabindex="0"
              type="submit"
            >
              Submit
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col p-2 max-w-md"
    >
      <form>
        <div
          class="flex flex-col items-start gap-4"
        >
          <div
            class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard css-1c2i806-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="true"
              for="outlined-basic"
              id="outlined-basic-label"
            >
              Max Bonus Credit
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall MuiInputBase-adornedStart css-l4u8b9-MuiInputBase-root-MuiInput-root"
            >
              <div
                class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-standard MuiInputAdornment-sizeSmall css-ittuaa-MuiInputAdornment-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-1pnmrwp-MuiTypography-root"
                >
                  $
                </p>
              </div>
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input MuiInputBase-inputSizeSmall MuiInputBase-inputAdornedStart css-nz481w-MuiInputBase-input-MuiInput-input"
                id="outlined-basic"
                name="maxOffer"
                type="number"
                value=""
              />
            </div>
          </div>
          <div
            class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard css-1c2i806-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="true"
              for="outlined-basic"
              id="outlined-basic-label"
            >
              Bonus match multiplier
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall MuiInputBase-adornedStart css-l4u8b9-MuiInputBase-root-MuiInput-root"
            >
              <div
                class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-standard MuiInputAdornment-sizeSmall css-ittuaa-MuiInputAdornment-root"
              >
                <span
                  class="notranslate"
                >
                  ​
                </span>
              </div>
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input MuiInputBase-inputSizeSmall MuiInputBase-inputAdornedStart css-nz481w-MuiInputBase-input-MuiInput-input"
                id="outlined-basic"
                name="matchMultiplier"
                type="number"
                value=""
              />
            </div>
          </div>
          <div
            class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-wb57ya-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-standard css-1c2i806-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="true"
              for="outlined-basic"
              id="outlined-basic-label"
            >
              Weekly Offers Chances
            </label>
            <div
              class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-sizeSmall MuiInputBase-adornedStart css-l4u8b9-MuiInputBase-root-MuiInput-root"
            >
              <div
                class="MuiInputAdornment-root MuiInputAdornment-positionStart MuiInputAdornment-standard MuiInputAdornment-sizeSmall css-ittuaa-MuiInputAdornment-root"
              >
                <span
                  class="notranslate"
                >
                  ​
                </span>
              </div>
              <input
                aria-invalid="false"
                class="MuiInputBase-input MuiInput-input MuiInputBase-inputSizeSmall MuiInputBase-inputAdornedStart css-nz481w-MuiInputBase-input-MuiInput-input"
                id="outlined-basic"
                name="weeklyOfferChance"
                type="number"
                value=""
              />
            </div>
          </div>
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
            name="submit"
            tabindex="0"
            type="submit"
          >
            Submit
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
